src/cargo/core/compiler/fingerprint.rs

```diff
@@ -1698,23 +1721,13 @@ where
     None
 }
 
-#[repr(u8)]
+#[derive(Serialize, Deserialize)]
```

Comment:
```suggestion
```

This doesn't seem necessary?

Reply:
Heh, a leftover from a previous iteration!